### PR TITLE
KIALI-1617 Add VS/DR validation list

### PR DESCRIPTION
**Pull request by xeviknal**
List all the validations performed by kiali on Destination Rules and Virtual Services.

screenshot of kiali 1

## Column description:
Validation message: Message that is shown in the ACEditor
Severity: error/warning
Description: broader description about the validation
Source: link to source code
Example: link to a yaml that triggers the validation explained

### CODE REVIEW

File: content/documentation/overview/index.adoc [hunk 1]
```diff
@@ -115,6 +116,65 @@ Kiali will highlight configuration errors.
 image::istio-yaml-validation.png[Demo Website]
 {nbsp} +
 
+==== Validations performed
+[cols="2,1,2,1,1", options="header"]
+.List of destination rule validations
```
Comment (abonas) [inline content/documentation/overview/index.adoc]: is this the list of Kiali specific validations? the ones that are *not* done by Galley? or list of all validations together?

Comment (xeviknal) [reply content/documentation/overview/index.adoc]: they are the one not done by Galley, the specific ones created in Kiali.

Comment (abonas) [reply content/documentation/overview/index.adoc]: I would specify this in the documentation so it is clear these are "extra" ones that do not exist in Istio

Comment (abonas) [reply content/documentation/overview/index.adoc]: other than that, it is LGTM

**Comment by xeviknal** [comment]
@lucasponce, I added the deprecated validations (same as kiali/kiali#1668)
@abonas I also added a little comment at the beginning of the section. Do you thing is clear enough?

File: content/documentation/overview/index.adoc [hunk 2]
```diff
@@ -115,6 +117,92 @@ Kiali will highlight configuration errors.
 image::istio-yaml-validation.png[Demo Website]
 {nbsp} +
 
+==== Validations performed
+
+In this section, there are listed all the validations that Kiali perform for all the Istio Configs. Most of these validations are extra to the existent ones performed by Galley (excepting those marked as deprecated).
```
Comment (abonas) [inline content/documentation/overview/index.adoc]: How about:
"This section lists all the validations that Kiali performs on all Istio configurations. Most of these validations are done in addition to/on top of the existing ones performed by Istio's Galley component (except those marked as deprecated)"

**Comment by xeviknal** [comment]
@abonas comments approached :) thanks for the fine-tuning!

**Pull request merged**